lib/core-integration/src/Test/Integration/Framework/DSL.hs

```diff
@@ -785,6 +786,27 @@ emptyWalletWith ctx (name, passphrase, addrPoolGap) = do
     expectResponseCode @IO HTTP.status201 r
     return (getFromResponse id r)
 
+rewardWallet :: Context t -> IO (ApiWallet, Mnemonic 24)
```

Comment:
why not to call it `readMIRwallet` to better reflect its nature?

Reply:
I was shared between calling everything `MIR` (which is sort of cardano-node specific terminology) or simply `Reward` (which is more generic). 

I ended up referring to "MIR" in the setup code, because it makes it clearer to what it connects behind the scene, but resorted to "reward" for everything in the DSL and scenarios. That the rewards come from MIR certificates is sort of an implementation detail.